src/pages/Lobby.tsx

```diff
@@ -329,23 +338,19 @@ const LobbyPage = ({ ws, currentUser }: LobbyProps): JSX.Element => {
         </div>
       </div>
 
-      {createGameModal && (
-        <div className="center-modal background-color-grey margin-top-30">
-          <CreateGameForm ws={ws} toggleOff={closeCreateGameModal} />
-        </div>
-      )}
+      {createGameModal && <CreateGameForm ws={ws} toggleOff={closeCreateGameModal} />}
```

Comment:
The height of this game form is a bit small if you remove the div, maybe make it bigger? The width and height seems unbalanced

Reply:
Div was moved into the GameLobby component, and height changed to 50vh.
For Create-Game modal, height of modal shrunk to only wrap children to remove extra white space
For Game Lobby, revert back to original height of 75vh.